src/widgets/_windows.scss

```
@@ -117,6 +117,23 @@ window:not(.popup):not(.menu) {
             text-shadow: none;
             -gtk-icon-shadow: none;
         }
+
+        pane-separator {
+            background-color: rgba(black, 0.0);
+            box-shadow: none;
+        }
+
+        separator {
+            background: none;
+            margin: 0 rem(18px);
+        }
+
+        separator:first-child,
+        separator:last-child {
```

If this is meant for the separators next to titlebuttons then a better selector would be `
```suggestion
        separator.titlebutton {
```